okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -323,7 +371,7 @@ public void shutdown() {
       if (frameWriter != null) {
         frameWriter.goAway(0, ErrorCode.NO_ERROR, new byte[0]);
       }
-      onGoAway(Integer.MAX_VALUE, Status.INTERNAL.withDescription("Transport stopped"));
+      onGoAway(Integer.MAX_VALUE, Status.UNAVAILABLE.withDescription("Transport stopped"));
```

Comment:
This is for consistency with the Netty implementation. IMO, this code makes more sense as the one to standardize on.